java-checks/src/test/files/checks/AccessibilityChangeCheck.java

```diff
@@ -0,0 +1,8 @@
+class A {
+    void makeItPublic(String methodName) throws NoSuchMethodException {
+      this.getClass().getMethod(methodName).setAccessible(true); // Noncompliant {{Remove this accessibility update.}}
```

Comment:
Could you add a test for `public static void setAccessible​(AccessibleObject[] array, boolean flag)`?